acl/policy.go

```diff
@@ -134,6 +134,12 @@ func Parse(rules string) (*Policy, error) {
 		return nil, fmt.Errorf("Failed to parse ACL Policy: %v", err)
 	}
 
+	// At least one valid policy must be specified, we don't want to store only
+	// raw data
+	if len(p.Namespaces) < 1 && p.Agent == nil && p.Node == nil && p.Operator == nil && p.Quota == nil {
```

Comment:
Can you create an empty method on the policy object and just call that here.

Also: `len(p.Namespaces) == 0 && ...`

Reply:
Sure, that would make this cleaner.